packages/gestalt/src/TableSortableHeaderCell.js

```diff
@@ -38,33 +38,36 @@ export default function TableSortableHeaderCell(props: Props) {
 
   return (
     <TableHeaderCell colSpan={colSpan} rowSpan={rowSpan} scope={scope}>
-      <Touchable
-        onTouch={onSortChange}
-        onMouseEnter={() => setHovered(true)}
-        onMouseLeave={() => setHovered(false)}
-        onFocus={() => setFocused(true)}
-        onBlur={() => setFocused(false)}
-      >
-        <Box display="flex">
-          {children}
-          <Box
-            marginLeft={2}
-            dangerouslySetInlineStyle={{
-              __style: { visibility },
-            }}
-          >
-            <Icon
-              accessibilityLabel=""
-              icon={
-                status === 'active' && sortOrder === 'asc'
-                  ? 'sort-ascending'
-                  : 'sort-descending'
-              }
-              color={status === 'active' ? 'darkGray' : 'gray'}
-            />
+      <Box width="fit-content">
```

Comment:
@PayalModi `fit-content` is not supported on IE 11. Instead let's try to use `display="inlineBlock"`.

https://caniuse.com/#search=fit-content

![image](https://user-images.githubusercontent.com/127199/83815452-1318d380-a675-11ea-9bda-0697256df907.png)